components/core/src/clp/ffi/ir_stream/decoding_methods.cpp

```
@@ -589,11 +590,12 @@ IRErrorCode get_encoding_type(ReaderInterface& reader, bool& is_four_bytes_encod
     return IRErrorCode_Success;
 }
 
-IRErrorCode deserialize_tag(ReaderInterface& reader, encoded_tag_t& tag) {
+auto deserialize_tag(ReaderInterface& reader, encoded_tag_t& tag)
```

Cppcheck (2.19.0)

[style] The function 'deserialize_tag' is never used.

(unusedFunction)